Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -1748,6 +1754,20 @@ static void webkit_web_view_class_init(WebKitWebViewClass* webViewClass)
 #endif
         WEBKIT_PARAM_READABLE);
 
+    /**
+     * WebKitWebView:has-immersive-session:
```

Comment:
This could have an introspection attribute to link the getter function to the property:

```suggestion
     * WebKitWebView:has-immersive-session: (attributes org.gtk.Property.get=webkit_web_view_get_has_immersive_session) (getter get_has_immersive_session):
```

(Yes, I know most of WebKit code lacks this kind of GI annotations, but that's because GTK-Doc&mdash;used in the past&mdash;didn't support them... it's always a good moment to start using them!)